### PR TITLE
Implemented unit tests for AdminUsersView

### LINKED ISSUE #469: Add unit tests for adminUsersView
## ✨ Feature request
I want to know ~~if~~ when I break stuff.

### 📜 Acceptance Criteria
- [ ]  There should be 50% code coverage on this component

### 💡 Additional context
https://lmiller1990.github.io/vue-testing-handbook/

### PR DESCRIPTION
**Pull request by te4-david-jensen**
## ⚙️ What issue did you implement or fix?
Fixes #469 

## 📜 Summary
- Implemented unit tests for AdminUsersView

## 📝 Type of change
- [x] ✨ New feature or enhancement (non-breaking change which adds functionality)

## ✅ Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

## 💩 How can this implementation be improved?
- Improve test coverage

### CODE REVIEW

**Review by Te4-Jacob-sandstrom — APPROVED** [review]

File: frontend/src/views/__tests__/AdminUsersView.spec.ts [hunk 1]
```diff
@@ -0,0 +1,16 @@
+import { shallowFactory } from '@/helpers/testFactoryHelpers';
+import { Wrapper } from '@vue/test-utils';
+import AdminUsersView from '../AdminUsersView.vue';
+import Vue from 'vue';
+
+describe('AdminUsersView.vue', () => {
+  const wrapper: Wrapper<Vue> = shallowFactory(AdminUsersView);
```
Comment (adr29truck) [inline frontend/src/views/__tests__/AdminUsersView.spec.ts:7]: You don't wan't a before do so that the wrapper is reset between all tests?

**Review by adr29truck — CHANGES_REQUESTED** [review]
Looks good. Just one comment

File: frontend/src/views/__tests__/AdminUsersView.spec.ts (hunk 1 above)
Comment (te4-adrian-almetun-smeds) [inline frontend/src/views/__tests__/AdminUsersView.spec.ts:7]: You don't want a before do so that the wrapper is reset between all tests?

**Review by te4-adrian-almetun-smeds — CHANGES_REQUESTED** [review]
Just a thought

File: frontend/src/views/__tests__/AdminUsersView.spec.ts (hunk 1 above)
Comment (te4-david-jensen) [reply frontend/src/views/__tests__/AdminUsersView.spec.ts:7]: I it necessary to reset the wrapper if you only ever read from it?

Comment (te4-adrian-almetun-smeds) [reply frontend/src/views/__tests__/AdminUsersView.spec.ts:7]: Currently I suppose your right we don't need it. But for futureproofing it, we could do use a before each.

**Review by te4-adrian-almetun-smeds — APPROVED** [review]
LGTM 👍

**Pull request merged**